src/WebUI/Utils.ts

```diff
@@ -25,4 +37,24 @@ export class Utils {
 
         return labelArray;
     }
+
+    public static generateEqualsConditionLabelSelector(labels: { [key: string]: string }): string {
+        console.log(labels);
+        let labelSelector: string = "";
+        if(labels) {
+            const keySet = Object.keys(labels);
+            keySet.forEach((key,index) => {
+                labelSelector = labelSelector.concat(format("{0}={1}", key, labels[key]))
+                if (index < keySet.length-1) labelSelector = labelSelector.concat(",");
+            });
+        }
+        return labelSelector;

```

Comment:
if we want a comma separated list, we can put the labels in to a string and finally can concat